dolphinscheduler-ui/src/views/projects/task/instance/use-table.ts

```diff
@@ -262,6 +262,7 @@ export function useTable() {
                         circle: true,
                         type: 'info',
                         size: 'small',
+                        disabled: !row.host,
```

Comment:
Can we provide message here? this may cause confusion, why the download button can not be clicked.

Reply:
I think download button should have the same display like view log button.